<!-- 		-->
<launch>
	<!-- 上位机给下位机发送消息命令 -->
	
	<node pkg="pc_stm32" type="talk_stm32" name="talker_node"  >

		<param name="car_goal" value="/car_goal" />
		<param name="pid_params" value="/pid_params" />
		<param name="port_name" value="/dev/stm" />
		<param name="baudrate" value="115200" />
		
	</node>
	

	
</launch>
